tests/unit/modal_test.coffee

```diff
@@ -0,0 +1,19 @@
+moduleForComponent 'modal', '[Unit] Modal component',
+  needs: [
+    'template:modal'
+    'template:modal_header'
+    'template:modal-footer'
+  ],
+  
+  teardown: ->
+    Ember.Widgets.DISABLE_ANIMATIONS = no
```

Comment:
It's already destroyed in the test, but that's not a good reason.I would hope that Ember QUnit does it for us. 

I'll check that.